extensions/blocks/anchor-fm/editor.js

```diff
@@ -14,6 +13,7 @@ import { registerPlugin } from '@wordpress/plugins';
 import { name } from '.';
 import getJetpackExtensionAvailability from '../../shared/get-jetpack-extension-availability';
 import { waitForEditor } from '../../shared/wait-for-editor';
+import buidlTemplate from './basic-template';
```

Comment:
Tiny edit, should this be `buildTemplate`?

Reply:
yes, thanks.